src/Groups/GAPGroups.jl

```diff
@@ -1930,7 +1930,7 @@ end
 
 
 @doc raw"""
-    map_word(g::FPGroupElem, genimgs::Vector; genimgs_inv::Vector = Vector(undef, length(genimgs)), init = nothing)
+    map_word(g::Union{FPGroupElem, SubFPGroupElem}, genimgs::Vector; genimgs_inv::Vector = Vector(undef, length(genimgs)), init = nothing)
```

Comment:
question: for sub-pc and sub-fp I have to specify images for the generators of the full group or for the generators of the sub group?